include/Einleitung.tex

```diff
@@ -1,3 +1,16 @@
 \chapter{Einleitung}
+In der kooperativen Speicherbibliothek werden Exemplare der dem Verein zugehörigen Bibliotheken eingelagert. Das heisst das Exemplar der Bibliothek wird gereinigt und inventarisiert eingelagert. Sollte ein Bibliotheksbenutzer ein Exemplar dieses Werks anfordern, wird das Exemplar entweder per Kurier an die jeweilige Bibliothek verschickt, oder bei Journalen oder Magazinen vorzugsweise, eingescannt und in digitaler Form an den Benutzer übergeben. Sowohl die beteiligten Bibliotheken, wie auch die Bibliotheksbenutzer sind daher an einer zeitgerechten Lieferung interessiert. Dies bedeutet für die Speicherbibliothek, dass alle Prozesse und Abläufe effizient und zuverlässig ablaufen müssen. Da Verzögerungen in Zwischenschritten sich auf die ganze Auslieferung des Exemplars oder der digitalen Kopie desjenigen, auswirken.
+
+Im Hochregallager der Speicherbibliothek werden momentan bis zu 110'000 Behältern mit verschiedenen Exemplaren (von welchen viele mit RFID Tags ausgestattet sind) gelagert. Die Behälter werden manuell von Menschen befüllt und anschliessend wird der Behälter voll autonom an einen Lagerplatz gefahren. Zeitweise können auch gewisse Exemplare wieder aus den Behältern entnommen werden um dieses Exemplar zu Lesen, Scannen oder einer der teilnehmenden Bibliotheken zurückzusenden. Während dem Vorgang des Lagerns und Entnehmen der Exemplare werden weiterhin Menschen für das Befüllen und Pflegen der Behälter verwendet. Dies birgt die Gefahr, dass eine Person aus Versehen ein Exemplar in einen falschen Behälter einsortiert. Und so das Exemplar nur sehr umständlich wiedergefunden werden kann \parencite{WickiBaumann2019Projektbeschrieb}.
+
+In dieser Arbeit wird daher eine Lösung für das Problem der Deplatzierung als Konzept ausgearbeitet, dieses Konzept in einer Machbarkeitsstudie geprüft und mit einer Referenzimplementation validiert.
 
 \section{Aufgabenstellung und Zielsetzung}
+Ziel dieser Arbeit ist es "mittels einer Machbarkeitsstudie und einem Proof of Concept untersucht werden wie technisch realisiert werden kann, solche mit RFID-ausgerüstete Einzelexemplare (Bücher, Zeitschriften, etc.) im vollautomatischen Behälter-Hochregallager mit bis zu 3.1 Mio. Exemplaren zu identifizieren und zu finden" \parencite{WickiBaumann2019Projektbeschrieb}.
```

Comment:
Ziel dieser Arbeit ist es, mittels einer Machbarkeitsstudie und einem Proof of Concept zu untersuchen, ob es technisch realisierbar ist, Exemplare, welche mit RFID Tags ausgerüstet sind im Hochregallager der Speicherbibliothek zu finden. _(Das geschriebene macht insofern keinen sinn, da textpassagen fehlen, welche die grammatikalische noch bräuchte)_